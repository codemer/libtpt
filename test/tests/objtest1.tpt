Object Test
@foreach (obj.fruit) { ${.} }

@foreach (obj.count) { ${.} }

@keys(k, obj.fruitid)\
@foreach (k) {
${.} = ${obj.fruitid.${.}}
}
