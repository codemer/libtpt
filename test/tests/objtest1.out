Object Test
Apple Orange Grapes Banana Lemon Lime Kiwi Peach Pear Nectarine 
0 1 2 3 4 5 6 7 8 9 
Apple = 0
Banana = 3
Grapes = 2
Kiwi = 6
Lemon = 4
Lime = 5
Nectarine = 9
Orange = 1
Peach = 7
Pear = 8
